nautobot_ssot/jobs/base.py

```diff
@@ -80,9 +80,13 @@ def calculate_diff(self):
         """
         if self.source_adapter is not None and self.target_adapter is not None:
             self.diff = self.source_adapter.diff_to(self.target_adapter, flags=self.diffsync_flags)
-            self.sync.diff = self.diff.dict()
             self.sync.summary = self.diff.summary()
             self.sync.save()
+            try:
+                self.sync.diff = self.diff.dict()
+                self.sync.save()
+            except Exception:  # pylint: disable=broad-except
+                self.log_warning(message="Unable to save JSON Diff to database")
```

Comment:
There is a save for all other data prior to this try/except, so unless there is something I am not aware of, the only penalty would be that the diff value would be left to default ({}) instead of having a diff representation.